packages/core/src/features/quickSearch/quickSearch.ts

```diff
@@ -1,97 +1,40 @@
-import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+import type { Middleware, Reducer } from "@reduxjs/toolkit";
+import { coreCreateApi } from "src/coreCreateApi";
 import { GDC_APP_API_AUTH } from "../../constants";
-import {
-  CoreDataSelectorResponse,
-  createUseCoreDataHook,
-  DataStatus,
-} from "../../dataAccess";
-import { CoreState } from "../../reducers";
-import { CoreDispatch } from "../../store";
+import serializeQueryArgsWithDataRelease from "src/serializeQueryArgs";
 
-export interface QuickSearchState {
+interface QuickSearchState {
   searchList: Array<Record<string, any>>;
   query: string;
-  status: DataStatus;
-  readonly requestId?: string;
 }
 
-const initialState: QuickSearchState = {
-  searchList: [],
-  query: "",
-  status: "uninitialized",
-};
-
-interface QuickSearchInterface {
-  data: {
-    query: {
-      hits: Array<Record<string, any>>;
-    };
-  };
-}
-
-export const fetchQuickSearch = createAsyncThunk<
-  QuickSearchInterface,
-  string,
-  { dispatch: CoreDispatch; state: CoreState }
->("quickSearch/fetchQuickSearch", async (searchString: string) => {
-  if (searchString.length > 0) {
-    const response = await fetch(
-      `${GDC_APP_API_AUTH}/quick_search?query=${searchString}&size=5`,
-    );
-    if (response.ok) {
-      return await response.json();
-    }
-
-    throw Error(await response.text());
+export const fetchQuickSearch = async (searchString: string) => {
+  const response = await fetch(
+    `${GDC_APP_API_AUTH}/quick_search?query=${searchString}&size=5`,
```

Comment:
should we make a new const variable as `MAX_SEARCH_ITEMS` or something?

Reply:
I agree. It would be good to have this as a constant, even if we never change it, is describe what that value is.